tests/GoogleFontsTest.php

```diff
@@ -7,7 +7,7 @@
 use function Spatie\Snapshots\assertMatchesHtmlSnapshot;
 
 it('loads google fonts', function () {
```

Comment:
Could you use [a Pest dataset here](https://pestphp.com/docs/datasets), to run the test with both the array and the string?